kubetest/kind/kind.go

```diff
@@ -383,13 +391,38 @@ func (d *Deployer) TestSetup() error {
 	return nil
 }
 
+// clusterExists checks if a kind cluster with 'name' exists
+func (d *Deployer) clusterExists() (bool, error) {
+	log.Printf("kind.go:clusterExists(): %s", d.kindClusterName)
+
+	cmd := exec.Command("kind")
+	cmd.Args = append(cmd.Args, []string{"get", "clusters", flagLogLevel}...)
+	out, err := d.control.Output(cmd)
+	if err != nil {
+		return false, err
+	}
+
+	if strings.Contains(string(out), d.kindClusterName) {
```

Comment:
the above is not ideal without line split, but let's get the rest working.

> pushed an image

thanks!

Reply:
check, also sooner rather than later I hope to switch us to kubetest2 :^)